pkg/trace/propagation_test.go

```diff
@@ -14,6 +14,40 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+func (suite) TestForceNoTracingByHeader(t *testing.T) {
+	defer app.SetName(app.Info().Name)
+	app.SetName("gobox")
+
+	recorder := tracetest.NewSpanRecorderWithOptions(tracetest.Options{
+		SamplePercent: 0.1,
```

Comment:
You should use `NewSpanRecorder` here without the options. That way you will use 100% sampling instead of 0.1%.